educational-teams-bot/src/app/features/reactions/components/reactions-table/reactions-table.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ReactionsTableComponent } from './reactions-table.component';
+
+describe('ReactionsTableComponent', () => {
+  let component: ReactionsTableComponent;
+  let fixture: ComponentFixture<ReactionsTableComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ ReactionsTableComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ReactionsTableComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
```

Comment:
Deja Vu!
I've just been in this place before
(Higher on the street)
And I know it's my time to go
Calling you, and the search is a mystery
(Standing on my feet)
It's so hard when I try to be me, woah
Whooooaaa!!